hack/generate-metallb-manifests.sh

```diff
@@ -45,3 +36,23 @@ ls -d config/crd/bases/* | grep -v metallb.io_metallbs | xargs rm
 cp -r "$METALLB_PATH"/config/crd/bases config/crd
 cp -r "$METALLB_PATH"/config/crd/crd-conversion-patch.yaml config/crd
 cp -r "$METALLB_PATH"/config/webhook config/webhook
+
+# generate metallb chart
+rm -rf "$METALLB_PATH"/charts/metallb/charts
+rm -f "$METALLB_PATH"/charts/metallb/templates/rbac.yaml
+rm -f "$METALLB_PATH"/charts/metallb/templates/service-accounts.yaml
+rm -f "$METALLB_PATH"/charts/metallb/templates/webhooks.yaml
+
+yq e --inplace 'del(."dependencies")' "$METALLB_PATH"/charts/metallb/Chart.yaml
+
+find "$METALLB_PATH"/charts/metallb -type f -exec sed -i -e 's/{{ template "metallb.fullname" . }}-//g' {} \;
+sed -i -e 's/app.kubernetes.io\///g' "$METALLB_PATH"/charts/metallb/templates/controller.yaml
+sed -i -e 's/metallb-webhook-service/webhook-service/g' "$METALLB_PATH"/charts/metallb/templates/controller.yaml
```

Comment:
why do we need this?

Reply:
I think it'd make sense to align the operator to metallb and use the new name

Reply:
Ignore what I wrote here (about the webhook service name), I thought it was a leftover of https://github.com/metallb/metallb/issues/1494 while it's really a matter of a difference between the charts and the manifests.